doc/api/https.markdown

```diff
@@ -195,7 +195,7 @@ Example:
       ...
     }
 
-Or does not use an `Agent`.
+Or don't use an `Agent`.
```

Comment:
I tried to keep it to the grammar fix only. When changing the wording, I'd rather use something along these lines:

> Alternatively, opt out of connection pooling.

That's how `agent: false` is documented above and describes the intent/effect instead of an implementation detail.